README.md

```diff
@@ -14,3 +14,8 @@ See [./docs/development-deployment.md](./docs/development-deployment.md)
 - Production firebase console: [here](https://console.firebase.google.com/u/0/project/dvp-landing-266011/overview)
 - Staging firebase console: [here](https://console.firebase.google.com/u/0/project/dvp-landing-staging/overview)
 - Staging2 firebase console: [here](https://console.firebase.google.com/u/0/project/dvp-landing-static/overview)
+
+## Analytics:
+Production analytics: [here](https://analytics.google.com/analytics/web/#/p220899285/reports/intelligenthome)
```

Comment:
If I go to that analytics link, then it take me to this Google Analytics sign up/account creation page:
![image](https://user-images.githubusercontent.com/8318425/235456601-80808552-b4d0-4278-b35a-c2bffe074059.png)

I think it would be better to use this link - https://console.firebase.google.com/project/dvp-landing-266011/analytics this goes directly to the analytics dashboard in firebase without having to go through Google Analytics. Like: 
![image](https://user-images.githubusercontent.com/8318425/235456761-61662b71-7785-4828-8254-ef293c97147b.png)

NOTE! That firebase analytics doesn't load for me if I have my adblocker (uBlock Origin) enabled (same as https://stackoverflow.com/a/75927495/5791411). We should add a *WARN* note just before the link re this.

Reply:
This has now been resolved.